__tests__/app.test.js

```diff
@@ -21,12 +20,52 @@ describe('/api/categories', () => {
             .expect(200)
             .then(({body}) => {
                 expect(Array.isArray(body.categories)).toBe(true)
-                expect(body.categories.length > 0)
+                expect(body.categories.length = 4)
```

Comment:
Not sure this is doing what you want it to be doing? Refactor to use .toBe(4)

Reply:
done